google-cloud-storage/lib/google/cloud/storage/policy.rb

```diff
@@ -309,25 +333,57 @@ def roles_to_gapi
       #
       #   bucket.policy requested_policy_version: 3 do |p|
       #     p.version # 3
-      #     p.bindings.push({
-      #                       role: "roles/storage.admin",
-      #                       members: ["user:owner@example.com"],
-      #                       condition: {
-      #                         title: "test-condition",
-      #                         description: "description of condition",
-      #                         expression: "expr1"
-      #                       }
-      #                     })
+      #     p.bindings.insert({
+      #                         role: "roles/storage.admin",
+      #                         members: ["user:owner@example.com"],
+      #                         condition: {
+      #                           title: "test-condition",
+      #                           description: "description of condition",
+      #                           expression: "expr1"
+      #                         }
+      #                       })
       #   end
       #
       class PolicyV3 < Policy
+        ##
+        # Returns the Policy's bindings as an array of hashes that associate roles with an
+        # array of members. Conditions are allowed in the hashes. See [Understanding
+        # Roles](https://cloud.google.com/iam/docs/understanding-roles) for a
+        # listing of primitive and curated roles. See [Buckets:
+        # setIamPolicy](https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy)
+        # for a listing of values and patterns for members.
+        #
+        # @raise [RuntimeError] If version is 1.
```

Comment:
This seems wrong.

Reply:
I moved this to a YARD @attr tag up above on line 301.